_posts/2016-03-16-AngularTS.md

```diff
@@ -10,28 +10,27 @@ comments: true
 
 ## Combining the best of two worlds.
 
-Since my introduction to the heroic AngularJS framework at Devoxx around 4 years ago.
-I was intrigued and set for an adventure.
+Since my introduction to the heroic AngularJS framework at Devoxx around 4 years ago, I was intrigued and set for an adventure.
 With the upcoming release of Angular 2 we have to prepare ourselves with the migrating road map coming up.
-One of the core changes in Angular was changing the code language to TypeScript.
+One of the core changes in Angular 2 is the focus on using TypeScript.
 This blog will cover the use of Angular components in TypeScript.
 But what is this TypeScript? 
-TypeScript is a superset of JavaScript that focuses on strong typing and the new ES6 features: classes, interfaces and modules.
+It is a superset of JavaScript that focuses on strong typing and new ES6 features: classes, interfaces and modules.
```

Comment:
I would rollback this and change it into "But what is it? TypeScript is a superset of..."